proposals/NNNN-ATF_test_reports_folder_structure.md

```diff
@@ -0,0 +1,36 @@
+# Test Reports Folder structure
+
+* Proposal: [SDL-NNNN](NNNN-ATF_test_reports_folder_structure.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: ATF
+
+## Introduction
+
+For now ATF reports folder have inconvenient nested struct. ATF creates struct of folders from relative path to script.
+
```

Comment:
@GetmanetsIrina 
- folder _has_

- It would be good to see the current folder structure and what exactly is inconvenient in order to understand the difference with proposed solution

- maybe it worth to name the proposal like "Change ATF test report folder structure" so it will reflect the sense more clear

Reply:
:+1: